_inc/client/at-a-glance/scan.jsx

```diff
@@ -173,6 +174,17 @@ class DashScan extends Component {
 											'You will be notified if we find one.' ),
 										feature: 'rewind',
 									} )
+
+								}
+								{
+									<Card
+										key="security-scanning"
+										className="jp-dash-item__manage-in-wpcom"
+										compact
+										href={ `https://dashboard.vaultpress.com/${ this.props.siteRawUrl }/security/` }
```

Comment:
`this.props.siteRawUrl` won't work here. The VaultPress dashboard uses a VaultPress site ID, an   integer that is only used for the VaultPress dashboard.

`site_id` is part of `data`, that is fetched with the `jetpack/v4/module/vaultpress/data` endpoint, so you should be able to grab it from with `vaultPressData`.

Reply:
Thank you. I did a new commit.